packages/next/src/build/output/log.ts

```diff
@@ -75,3 +75,12 @@ export function warnOnce(...message: any[]) {
     warn(...message)
   }
 }
+
+export function timestamp() {
+  const now = new Date()
+  const hours = now.getHours().toString().padStart(2, '0')
+  const minutes = now.getMinutes().toString().padStart(2, '0')
+  const seconds = now.getSeconds().toString().padStart(2, '0')
+  const milliseconds = now.getMilliseconds().toString().padStart(3, '0')
+  return `${hours}:${minutes}:${seconds}:${milliseconds}`
```

Comment:
I think it is more common to use a dot to separate seconds and milliseconds.

```suggestion
  return `${hours}:${minutes}:${seconds}.${milliseconds}`
```

Reply:
We decided not to add timestamps. Most services putting out these logs will have timestamps already.